examples/README.md

```diff
@@ -186,3 +186,32 @@ kubectl port-forward jupyter-notebook-server 8888:8888
 # clean up
 kubectl delete -f ./examples/jupyter/jupyter-notebook-server.yaml
 ```
+
+## Jupyter Notebook Example (no experimental read cache)
+
+#### Prerequisite
+
+Your node pool must have created an ephemeral local ssds as described in
+https://cloud.google.com/kubernetes-engine/docs/how-to/persistent-volumes/local-ssd#node-pool
+
+#### Steps
+
+```bash
+# 1. replace <bucket-name> with your pre-provisioned GCS bucket name
+GCS_BUCKET_NAME=your-bucket-name
+sed -i "s/<bucket-name>/$GCS_BUCKET_NAME/g" ./examples/jupyter/jupyter-experimental-readcache.yaml
+
+# 2. install a Jupyter Notebook server using experimental gcsfuse read cache
+kubectl apply -f ./examples/jupyter/jupyter-experimental-readcache.yaml
+
+# get service IPs
```

Comment:
Could you also add the sequence number for this and the following steps? Or just remove the numbers 1, 2 from the above steps?

Reply:
Woops, yup, fixed! Thanks